lightning/src/ln/offers_tests.rs

```diff
@@ -1598,23 +1599,23 @@ fn fails_creating_or_paying_for_offer_without_connected_peers() {
 	let absolute_expiry = alice.node.duration_since_epoch() + MAX_SHORT_LIVED_RELATIVE_EXPIRY;
 	match alice.node.create_offer_builder(Some(absolute_expiry)) {
 		Ok(_) => panic!("Expected error"),
-		Err(e) => assert_eq!(e, Bolt12SemanticError::MissingPaths),
+		Err(e) => assert_eq!(e, Bolt12CreationError::BlindedPathCreationFailed),
 	}
 
 	let mut args = ReconnectArgs::new(alice, bob);
 	args.send_channel_ready = (true, true);
 	reconnect_nodes(args);
 
 	let offer = alice.node
-		.create_offer_builder(Some(absolute_expiry)).unwrap()
-		.amount_msats(10_000_000)
-		.build().unwrap();
+		.create_offer_builder(None).unwrap()
+		.amount(Amount::Currency { iso4217_code: *b"USD", amount: 6_000 })
```

Comment:
Since we are creating offers in fiat denomination in [Add tests for Offers with fiat amount](https://github.com/lightningdevkit/rust-lightning/pull/3175/commits/eee5eeb1a2b4ddc6a5198fe527a306cbeb22e22c) commit, maybe we can remove this change and the change below to simplify the PR?